client/posts/services/posts.service.coffee

```diff
@@ -1,12 +1,22 @@
 postsServ = ($meteor) ->
-  add: (newPost) ->
-      $meteor.call 'addPost', newPost
-  delete: (post) ->
-      $meteor.call 'deletePost', post.id
-  update: (post) ->
-      $meteor.call 'updatePost', post.id, post.text
-  findOwner: (post) ->
-      $meteor.object(Meteor.users, post.userId, false).subscribe('usersBasicInfo')
+    add: (newPost, image) ->
+        unless image.name == ''
+            image = Images.insert image, (error, fileObj) ->
+                if (error)
+                    alert error
+
+            # nie udało się przesłać pliku - nie wysyłaj samej treści
+            unless image? && image._id?
```

Comment:
alerts.service jest na developie
może najpierw to domergować i uzupełnić potem?